cmd/prometheus/main.go

```diff
@@ -67,6 +66,8 @@ import (
 	"github.com/prometheus/prometheus/tsdb"
 	"github.com/prometheus/prometheus/util/strutil"
 	"github.com/prometheus/prometheus/web"
+
+	"go.uber.org/atomic"
```

Comment:
This should be with the other non `prometheus/prometheus` imports above (same for remaining files)

Reply:
Sure 👍  

Is this verified by `goimports` on CI?

Reply:
> Is this verified by goimports on CI?

goimports would just sort the imports placed together AFAIK. Grouping relevant imports is on us.